tests/functionaltests/Tests/CortanaTests.mm

```diff
@@ -139,24 +139,24 @@ - (id)init {
 
 - (BOOL)application:(UIApplication*)application willFinishLaunchingWithOptions:(NSDictionary*)launchOptions {
     ASSERT_TRUE(launchOptions[UIApplicationLaunchOptionsVoiceCommandKey]);
-    WMSSpeechRecognitionResult* result = launchOptions[UIApplicationLaunchOptionsVoiceCommandKey];
-    ASSERT_STREQ("CORTANA_TEST", [result.text UTF8String]);
+    WAAVoiceCommandActivatedEventArgs* args = launchOptions[UIApplicationLaunchOptionsVoiceCommandKey];
+    ASSERT_STREQ("CORTANA_TEST", [args.result.text UTF8String]);
     _methodsCalled[NSStringFromSelector(_cmd)] = @(YES);
     return true;
 }
 
 - (BOOL)application:(UIApplication*)application didFinishLaunchingWithOptions:(NSDictionary*)launchOptions {
     ASSERT_TRUE(launchOptions[UIApplicationLaunchOptionsVoiceCommandKey]);
-    WMSSpeechRecognitionResult* result = launchOptions[UIApplicationLaunchOptionsVoiceCommandKey];
-    ASSERT_STREQ("CORTANA_TEST", [result.text UTF8String]);
+    WAAVoiceCommandActivatedEventArgs* args = launchOptions[UIApplicationLaunchOptionsVoiceCommandKey];
+    ASSERT_STREQ("CORTANA_TEST", [args.result.text UTF8String]);
     _methodsCalled[NSStringFromSelector(_cmd)] = @(YES);
     return true;
 }
 
-- (BOOL)application:(UIApplication*)application didReceiveVoiceCommand:(WMSSpeechRecognitionResult*)result {
+- (BOOL)application:(UIApplication*)application didReceiveVoiceCommand:(WAAVoiceCommandActivatedEventArgs*)args {
     // Delegate method should only be called once
```

Comment:
ToastNotificationsActivatedEventArgs has a String - argument and ValueSet - UserInput, so we can't just pass along the input value like we did with VoiceCommandActivatedEventArgs' SpeechRecognitionResult or ProtocolActivatedEventArgs' URI, and it made the most sense to pass along the projection through the delegate.  To keep consistency through the activation delegates, I changed them to also pass the entire args.  

---
In reply to: [70538277](https://github.com/Microsoft/WinObjC/pull/637#discussion_r70538277) [](ancestors = 70538277)